js_modules/dagit/packages/core/src/runs/RunMetadataProvider.tsx

```diff
@@ -109,14 +109,11 @@ function extractMetadataFromRun(run?: RunFragment): IRunMetadataDict {
   if (!run) {
     return metadata;
   }
-  if (run.stats.__typename !== 'RunStatsSnapshot') {
-    return metadata;
-  }
-  if (run.stats.startTime) {
-    metadata.startedPipelineAt = fromTimestamp(run.stats.startTime);
+  if (run.startTime) {
+    metadata.startedPipelineAt = fromTimestamp(run.startTime);
```

Comment:
At some point it'd probably be nice to propagate these name changes further into dagit and just keep startTime instead of `startedPipelineAt`, but we can do that later!